graylog-project-parent/pom.xml

```diff
@@ -416,6 +417,17 @@
                 <scope>provided</scope>
             </dependency>
             <dependency>
+                <groupId>com.gabrielittner.auto.value</groupId>
+                <artifactId>auto-value-extension-util</artifactId>
+                <version>${auto-value-extension-util.version}</version>
```

Comment:
The latter, it's a `compile` dependency for `auto-value-javabean` but will not be pulled in transitively in graylog2-server, because `auto-value-javabean` is pulled in as `provided` there.